website/docs/r/ingress.html.markdown

```diff
@@ -99,7 +99,7 @@ The following arguments are supported:
 
 * `metadata` - (Required) Standard ingress's metadata. For more info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#metadata
 * `spec` - (Required) Spec defines the behavior of a ingress. https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
-* `wait_for_load_balancer` - (Optional) Terraform will wait for the load balancer to have at least 1 endpoint before considering the resource created.
+* `wait_for_load_balancer` - (Optional) Terraform will wait for the load balancer to have at least 1 endpoint before considering the resource created. Defaults to `false`.
```

Comment:
I realized that it didn't make sense to default this to true for Ingress as we don't always know if there's going to be a load balancer associated with the ingress.

Reply:
I saw this error in the acceptance test, which might be related:

```
------- Stdout: -------
=== RUN   TestAccKubernetesIngress_InternalKey
    TestAccKubernetesIngress_InternalKey: resource_kubernetes_ingress_test.go:117: Step 1/3 error: Error running apply: 
        Error: Load Balancer is not ready yet     
```